Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -55,12 +60,20 @@ void LoadBankFromDiskTask::loadPulseTimes(::NeXus::File &file) {
   } catch (::NeXus::Exception &) {
     // Field not found error is most likely.
     // Use the "proton_charge" das logs.
-    thisBankPulseTimes = m_loader.alg->m_allBanksPulseTimes;
+    thisBankPulseTimes = m_loader.m_loadAlgorithm->m_allBanksPulseTimes;
     return;
   }
   std::string thisStartTime;
   size_t thisNumPulses = 0;
-  file.getAttr("offset", thisStartTime);
+  if (file.hasAttr("offset"))
+    file.getAttr("offset", thisStartTime);
+  else if (file.hasAttr("start"))
+    file.getAttr("start", thisStartTime);
+  else {
+    auto epoch = DateAndTime(DateAndTime::UNIX_EPOCH);
```

Comment:
Are there examples where either `offset` or `start` are not defined? Should we `throw` instead as I feel like the file is broken if we don't know the start time...

Reply:
👍 will look into this

Reply:
Hi Martyn,

This from Matt Jones
```
Hi Lamar, the ESS file writer always records a start time, and I believe the same is true of the ICP at ISIS. Does that answer your question?





9:23
It is probably possible to engineer a file with Mantid that does not contain a start time, but all real, raw data files have one AFAIK.
```

Reply:
@martyngigg I could log an error. I am not sure if there is any exception handling for this as it is a task which may be run in several threads.

Reply:
@LamarMoore Thanks. I think this point is probably too low level to log/throw an exception. We have a check in `LoadEventNexus` (commented on below). As Matt says if we know all files have offsets/start time should we keep throw in the `LoadEventNexus` part if the file doesn't contain it. Or maybe at least a warning that times have been set relative to Unix EPOCH?